inventory/group_vars/log

```diff
@@ -72,20 +72,21 @@ bonnyci_logs_apache_vhosts:
 
       ProxyPass /htmlify/ uwsgi://localhost:3310/
 
-      ServerSignature Off
-  - name: ara
-    vhost_extra: |
-      <Directory /var/www/ara>
-        Require all granted
-      </Directory>
-
-      ErrorLog /var/log/apache2/ara.log
-      CustomLog /var/log/apache2/ara_access.log combined
-
-      SetEnv ANSIBLE_CONFIG /var/www/ara/ansible.cfg
       Redirect permanent /ara /ara/
       ProxyPass "/ara/" uwsgi://localhost:3311/
 
+      <Location "/ara/">
+        ErrorLog /var/log/apache2/ara.log
```

Comment:
This isn't allowed inside a Location directive.
```text
May 05 22:34:43 logs apache2[31418]: Output of config test was:
May 05 22:34:43 logs apache2[31418]: AH00526: Syntax error on line 75 of /etc/apache2/sites-enabled/logs.conf:
May 05 22:34:43 logs apache2[31418]: ErrorLog not allowed here
May 05 22:34:43 logs apache2[31418]: Action 'configtest' failed.
```

It looks like if these share a vhost, they also need to share a log file.